src/main/java/spireMapOverhaul/zones/gremlinTown/monsters/GremlinAssassin.java

```diff
@@ -0,0 +1,112 @@
+package spireMapOverhaul.zones.gremlinTown.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.animations.AnimateSlowAttackAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.RollMoveAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.gremlinTown.powers.SapPower;
+import spireMapOverhaul.zones.invasion.powers.DrawReductionSingleTurnPower;
+
+import static spireMapOverhaul.util.Wiz.*;
+
+public class GremlinAssassin extends CustomMonster
```

Comment:
I have changed the gremlin gang v2 fight to be 3 gremlins instead of 4.  There aren't any bonus rewards but the damage taken tends to be more along the lines of act 2 hallway fights now.